tutorials/interoperability/using-axelar-sdk.md

```diff
@@ -0,0 +1,226 @@
+
+```bash
+npx hardhat compile
+```
+
+## Deploying the Cross-Chain Contract to Moonbase Alpha {: #deploying-the-cross-chain-contract-to-moonbase-alpha} 
+
+This demo focuses on using the scripts, so it’s best to take a look at them, starting with `deploy.js`, which is similar to example Ethers.js deployment contracts.
+
+`gatewayGasReceiver.js` stores many of the contract addresses in this repo, which are necessary for the deployment. You likely will not have to change any of the hardcoded addresses. Try deploying your contract to the origin chain:
+
+```bash
+npx hardhat run scripts/deploy.js --network moonbase
+```
+
+You should see the address deployed and printed in the console. Be sure to copy it! You will need it for interacting with the next script. You also need to deploy it to the destination chain. The choice of which destination network to use is up to you, but you will need its native currency to deploy. I’ve included some of the available networks and their faucets here:
+
+|                                 Network                                  |                   Deployment Command                   |
+|:------------------------------------------------------------------------:|:------------------------------------------------------:|
+|       [Sepolia](https://sepolia-faucet.pk910.de/){target=\_blank}        | `npx hardhat run scripts/deploy.js --network ethereum` |
+|           [Polygon Mumbai](https://faucet.polygon.technology/)           |  `npx hardhat run scripts/deploy.js --network mumbai`  |
+| [Avalanche Fuji](https://core.app/tools/testnet-faucet/){target=\_blank} |   `npx hardhat run scripts/deploy.js --network fuji`   |
+|     [Fantom Testnet](https://faucet.fantom.network/){target=\_blank}     |  `npx hardhat run scripts/deploy.js --network fantom`  |
+
+After running a deployment command, you'll see output like the below. Be sure to copy and destination chain's contract address, because you'll need to provide that in a later step. 
+
+--8<-- 'code/tutorials/interoperability/axelar-sdk/terminal/deploy.md'
+
+## Building the Mint.js Script {: #building-the-mint-js-script}
+
+The minting contract is quite exciting, and will require Axelar’s SDK. At the top of the script, Ethers.js is initialized in a Hardhat script. The Axelar SDK is also initialized. There are multiple Axelar APIs available in the SDK, but in this case we will only be using the AxelarQueryAPI sinceit includes all of the gas estimation functionality that we’ll need for paying gas fees across chains. 
+
+```js
+--8<-- 'code/tutorials/interoperability/axelar-sdk/mint-1.js'
+```
+
+There are also some constants for you to change right after. This walkthrough is using Fantom as the destination chain, but you can use whichever chains you deployed to. Note that even though we’re using a testnet environment, the chain names are still their mainnet equivalents, hence why the origin chain is “MOONBEAM” and not “MOONBASE”.
```

Comment:
is this referring to `const ORIGIN_CHAIN = 'moonbeam'`; - if so we should be referring to it as `moonbeam` and `moonbase` instead of using caps and quotes